tools/strip-units/_function.scss

```diff
@@ -4,12 +4,27 @@ section: Tools
 
 Removes the unit of number.
 
+**Use**
 ```scss
 .my-class {
   prop: strip-units(<number>);
 }
 ```
 
+**Use with a prop**
+```scss
+.my-class {
+  line-height: strip-units(2rem);
+}
+```
+
+**CSS Output**
+```css
+.my-class {
+  line-height: 2;
```

Comment:
Again, line-height was the only property I could think that you might not want units for? Not sure if this is a good example.

Reply:
This is handy more for maths in a mixin/function than to use in a CSS property. Instances could be you need to check to values of two numbers, one is a `em` and the other is a `rem`. Sass doesn’t allow maths of differing units, so this is a way to remove the units to accomplish the math.